nucliadb/src/migrations/0024_assign_models_to_kbs.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2021 Bosutech XXI S.L.
+#
+# nucliadb is offered under the AGPL v3.0 and as commercial software.
+# For commercial licensing, contact us at info@nuclia.com.
+#
+# AGPL:
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""Migration #24
+
+Vectorsets are coming and we need to be ready at nucliadb. Vector index config
+shouldn't be stored anymore in the `Shards` protobuffer, we need to migrate to
```

Comment:
I don't see you cleaning the shards protobuffer. Is this intended? Do you plan to clean at a later date?

Reply:
Yeah, just in case, we can clean that later